neo/Ledger/Blockchain.cs

```diff
@@ -345,7 +347,7 @@ private RelayResultReason OnNewConsensus(ConsensusPayload payload)
 
         private void OnNewHeaders(Header[] headers)
         {
-            using (Snapshot snapshot = GetSnapshot())
+            using (SnapshotView snapshot = GetSnapshot())
```

Comment:
If we use `var` keyword, we can change the name in the future without alter other lines

Reply:
I don't like `var` because it makes it impossible for me to visually see the type of variable.